projects/commudle-admin/src/app/app.component.scss

```diff
@@ -339,3 +339,13 @@
     }
   }
 }
+
+:host::ng-deep {
+  .cdk-overlay-container {
+    nb-card-header {
+      button {
+        display: none;
```

Comment:
This is affecting the join community dialog box and follow expert dialog box in the home page, as much as possible, let's avoid the `:host` selector